Sources/PointFree/Blog/BlogPostIndex.swift

```diff
@@ -30,30 +30,81 @@ let blogIndexMiddleware: (Conn<StatusLineOpen, Tuple3<Database.User?, Subscriber
     }
 )
 
-private let blogIndexView = View<(Database.User?, SubscriberState)> { currentUser, subscriberState in
+private let blogIndexView = View<(Database.User?, SubscriberState)> { currentUser, subscriberState -> Node in
 
+  let allPosts = Current.blogPosts()
+    .sorted(by: their(^\.id, >))
+  let newPosts = allPosts.prefix(3)
+  let oldPosts = allPosts.dropFirst(3)
```

Comment:
we may want to more aggressively collapse posts. maybe everything after the first?

Reply:
It's interesting...over time I don't think the historical posts matter in this ordinal way, but I think it's fine for now.